src/services/data-provider-link-parser.ts

```diff
@@ -26,6 +26,15 @@ export const getAllGroupTokens = async (): Promise<string[]> => {
   return groupTokens;
 };
 
+function doesURLMatch(projectUrl: string, path: string, name: string) {
+  const parsedUrl = parse(projectUrl);
+  const urlPath = parsedUrl.pathname;
+  const pathItems = urlPath.split('/');
```

Comment:
Can any of these items (such as `parsedUrl.pathname`) ever be null, or does that already throw a parsing exception?

Reply:
I think `parsedUrl.pathname` would just be an empty string if `parsedUrl` doesn't have a `pathName`, and I think nothing in this block should throw an exception. There is similar parsing code in `extractProjectInformation` a few lines above